docs/octopus_deploy_environment.md

```diff
@@ -0,0 +1,33 @@
+# octopus_deploy_environment
+
+Models/Tests an Octopus Deploy [Environment](https://octopus.com/docs/infrastructure/environments) resource
```

Comment:
```suggestion
Models/Tests an Octopus Deploy [Environment](https://octopus.com/docs/infrastructure/environments) resource.
```

Is this the same wording you've used elsewhere?

Reply:
It really isn't. I'll fix that